File: BeerProject/app/beer-library/src/app/models/beer.ts
import { Glass } from './glass';
import { Srm } from './srm';
import { Style } from './style'

export class Beer {
    public id: string;
    public name: string;
    public nameDisplay: string;
    public description: string;
    public abv: string;
    public glasswareId: number;
    public srmId: number;
    public styleId: number;
    public isOrganic: string;
    public status: string;
    public statusDisplay: string;
    public createDate: string;
    public updateDate: string;
    public glass: Glass;
    public srm: Srm;
    public style: Style;
    public showDetail: boolean; 
}

File: BeerProject/Models/Beer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BeerProject.Models
{
	public class Beer
	{
		public string id { get; set; }
		public string name { get; set; }
		public string nameDisplay { get; set; }
		public string description { get; set; }
		public string abv { get; set; }
		public int glasswareId { get; set; }
		public int srmId { get; set; }
		public int styleId { get; set; }
		public string isOrganic { get; set; }
		public string status { get; set; }
		public string statusDisplay { get; set; }
		public string createDate { get; set; }
		public string updateDate { get; set; }
		public Glass glass { get; set; }
		public Srm srm { get; set; }
		public Style style { get; set; }
	}
}